Dockerfile

```diff
@@ -495,6 +495,64 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-go as sdk-govc
+
+USER root
+RUN \
+  mkdir -p /usr/share/licenses/govmomi && \
+  chown -R builder:builder /usr/share/licenses/govmomi
+
+ARG ARCH
+ARG TARGET="${ARCH}-bottlerocket-linux-gnu"
+ARG GOVMOMIVER="0.26.0"
+ARG GOVMOMISHORTCOMMIT="34586b6"
+ARG GOROOT="/home/builder/go"
+ARG GOPATH="${GOROOT}/go"
+
+USER builder
+WORKDIR /home/builder
+COPY ./hashes/govmomi ./hashes
+RUN \
+  sdk-fetch hashes && \
+  tar xf govmomi-${GOVMOMIVER}.tar.gz && \
+  rm govmomi-${GOVMOMIVER}.tar.gz && \
+  mkdir -p ${GOPATH}/src/github.com/vmware && \
+  mv govmomi-${GOVMOMIVER} ${GOPATH}/src/github.com/vmware/govmomi && \
+  ln -s ${GOPATH}/src/github.com/vmware/govmomi
+
+ARG GOOS="linux"
+ARG CGO_ENABLED=0
+ARG PKGPATH="github.com/vmware/govmomi/govc/flags"
+ARG GOARCH_aarch64="arm64"
+ARG GOARCH_x86_64="amd64"
+ARG GOARCH_ARCH="GOARCH_${ARCH}"
```

Comment:
I don't see the need for these to be passed by users as `--build-arg`, can you switch these to use `ENV` instead of `ARG`?

Reply:
I think the only one should consider exposing to users is `GOOS` which would allow building the toolchain on other platforms - but even that isn't terribly helpful as the SDK isn't set up to build for anything other than the equivalent to `GOOS=linux` at this time.

Having the likes of `GOARCH_x86_64` exposed to users permits misconfiguration and confusion:

```bash
# Build on x86_64 for RISC-V 64
[[ "$(uname -om)" = "x86_64 GNU/Linux" ]] &&
  docker build -t my-sdk:build --build-arg GOOS=freebsd --build-arg GOARCH_x86_64=riscv64 .
```

I'm not sure we'll have a case where overriding _some_ of these will make a future change easier or harder. I suggest that we hide `CGO_ENABLED`, `PKGPATH`, and `GOARCH_*`. Maybe also `GOOS`, though I'm not convinced either way yet.